.circleci/marvin-reporter.js

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env node
+
+const execSync = require('child_process').execSync;
```

Comment:
have you considered using the circleci Slack Orb instead?

Reply:
This would mean replacing a single `curl` command inside the script (all the rest is required logic that will remain anyway) with a call to an Orb I know nothing about.
I prefer using curl. This is also much easier to test & debug locally from command line.